package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -32,10 +33,26 @@
 from kedro.io import DataCatalog
 from kedro.io.core import get_filepath_str
 from kedro.pipeline import Pipeline
-from semver import VersionInfo
+
+from kedro_viz.constants import KEDRO_VERSION
 
 logger = logging.getLogger(__name__)
-KEDRO_VERSION = VersionInfo.parse(__version__)
+
+
+class _VizNullPluginManager:
+    """This class creates an empty ``hook_manager`` that will ignore all calls to hooks
+    and registered plugins allowing the runner to function if no ``hook_manager``
+    has been instantiated.
+    """
+
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def __getattr__(self, name):
+        return self
+
+    def __call__(self, *args, **kwargs):
+        pass
```

Comment:
Is this needed? What's the different between this and the `NullPluginManager`.

Reply:
No difference really but - _NullPluginManager() was not introduced in kedro 0.17.5 which is the oldest kedro version that viz supports. So I had to create this.